runners/core-java/src/main/java/org/apache/beam/runners/core/UnboundedReadFromBoundedSource.java

```diff
@@ -58,16 +57,28 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.Iterator;
 import java.util.List;
 import java.util.NoSuchElementException;
 
 import javax.annotation.Nullable;
 
 /**
- * {@link PTransform} that performs a unbounded read from an {@link BoundedSource}.
+ * {@link PTransform} that converts a {@link BoundedSource} as an {@link UnboundedSource}.
+ *
+ * <p>{@link BoundedSource} is read directly without calling {@link BoundedSource#splitIntoBundles},
+ * and element timestamps are propagated. While any elements remain, the watermark is the beginning
+ * of time {@code BoundedWindow.TIMESTAMP_MIN_VALUE}, and after all elements have been produced
+ * the watermark goes to the end of time {@code BoundedWindow.TIMESTAMP_MAX_VALUE}.
```

Comment:
is this correct? Shouldn't it be `{@link BoundedWindow#TIMESTAMP_MIN_VALUE}`? I think that `.` does not work as a link there.